### DISCUSSION TITLE
Non-aggregated time feature extraction

### DISCUSSION THREAD

**Discussion by ytzfhqs** [blue-yonder/tsfresh#999] [Q&A] [answered]
I now have the following data:
date	                v1	v2	Target
2018/1/1 0:00	2.2	1.3	244033.7682
2018/1/1 0:15	2.4	1.5	232720.6402
2018/1/1 0:30	2.7	1.7	231685.6891
2018/1/1 0:45	2.5	1.9	227581.3317
2018/1/1 1:00	2.6	2.3	233482.4269
2018/1/1 1:15	2.2	1.4	224464.7554
2018/1/1 1:30	2.6	1.7	223881.3222
2018/1/1 1:45	2.7	1.6	221522.7156
date is the time column, v1, v2 are the feature columns, and Target is the column to be predicted。

I would like to know if tsfresh can help me automatically build features to better predict the Target column, which is a regression problem

**Comment by nils-braun** [comment, marked as answer]
Yes, you can use `tsfresh` to build features for this use-case. We even have an example for this: https://github.com/blue-yonder/tsfresh/blob/main/notebooks/05%20Timeseries%20Forecasting.ipynb
